### PR TITLE
Fix Paranoia to work with validates_uniquness_of (rails4 branch)

### PR DESCRIPTION
**Pull request by codeodor**
For the rails4 branch, in pursuit of #114

### CODE REVIEW

**Comment by codeodor** [comment]
@radar just wondering if you had a chance to look at this PR and the one in #121 too. I got bit by this again, so it reminded me of these. :smile:

**Comment by ethangunderson** [comment]
Any update on this one? I never learn, so this keeps biting us.

**Comment by radar** [comment]
If there was an update you can be sure it would be posted here. There is no secret place where things about this gem get discussed. It's all done here out in the open.

**Comment by ethangunderson** [comment]
I apologize, that was a poorly worded comment on my part. Let me try again. I would like to see this merged, is there anything I can do to help it along?

**Comment by codeodor** [comment]
This one should be ready now too.

**Comment by codeodor** [comment]
This will also fix #175
